doc/news/changes/major/20180709Harper

```diff
@@ -0,0 +1,8 @@
+New: FE_BernardiRaugel is the implementation of an element developed by Christine Bernardi and
+Genevieve Raugel in the 1985 Mathematics of Computation paper. The Bernardi-Raugel element is an
+enrichment of the P1-P0 element for Stokes problems by the addition of edge-based quadratic
+bubble functions. The original description is for simplicial meshes, but this implementation is
+for quadrilateral and hexahedral meshes. The BR1-P0 pair is LBB stable for Stokes problems. Due
+to the edge-based bubble functions, features such as hanging nodes are not supported.
+<br>
+(Graham Harper 07/09/2018)
```

Comment:
you need a newline here

Reply:
Where is the newline needed? A <br> tag or an actual newline in the file?

Reply:
Each file needs to end in a newline. You can see that it is missing if you look at it here: https://github.com/dealii/dealii/pull/6904/files

Reply:
Oh okay I see it now. Fixed it!